src/shmem_synchronization.h

```diff
@@ -47,6 +49,8 @@ shmem_internal_fence(shmem_ctx_t ctx)
 #ifdef USE_XPMEM
     ret = shmem_transport_xpmem_fence();
     if (0 != ret) { RAISE_ERROR(ret); }
+#else
+    shmem_internal_membar_store();
 #endif
```

Comment:
Same comment as quiet.  The shmem_transport_xpmem_fence routine is now redundant.